pkg/asset/ignition/bootstrap/baremetal/template.go

```diff
@@ -95,7 +99,11 @@ func GetTemplateData(config *baremetal.Platform, networks []types.MachineNetwork
 		}
 		templateData.ProvisioningDHCPAllowList = strings.Join(dhcpAllowList, " ")
 	case baremetal.DisabledProvisioningNetwork:
-		templateData.ProvisioningInterface = "ens3"
+		if config.BootstrapProvisioningInterfaceOverride == "" {
+			templateData.ProvisioningInterface = "ens3"
+		} else {
+			templateData.ProvisioningInterface = config.BootstrapProvisioningInterfaceOverride
```

Comment:
This nic is defined in the (installer created) bootstrap VM - so I'm not sure it's reasonable to expect the user to somehow know what the appropriate nic name is here?

I'm wondering why the VM nic name is different in your environment (which is ARM I believe) - is there some way we can make the nic name consistent between architectures instead, or else detect the required value based on the architecture?